src/markdown-pages/docs/cli-command-reference/runs-submit.md

```diff
@@ -13,23 +13,27 @@ The following section provides a subset of examples of how you can use the `runs
 
 ### Getting help
 
-Use the following commands to get more information about the command and command options, including default values.
+Use the following commands to get more information about the command and command options on Mac or Unix, including default values.
 
 ```
 galasactl --help
-galasactl runs --help
-galasactl runs runs --help
+```
+
+If you are using Windows, use the following commands:
+
+```
+galasactl.exe --help 
 ```
 
 ### Submitting tests from a portfolio
```

Comment:
> Submitting tests from a portfolio 

Should be `Submitting tests to an ecosystem from a portfolio` ?

Reply:
updated